### PR TITLE
add --api-versions flag

### PR DESCRIPTION
**Pull request by vivian-src**
Part-of: [AB#3334](https://dev.azure.com/syncier/cloud/_workitems/edit/3334)
- added apiVersions list for helm-chart
- added test case

### CODE REVIEW

File: pkg/helmt/helmt.go [hunk 1]
```diff
@@ -124,6 +125,11 @@ func template(name string, chart string, values []string, namespace string, skip
 	} else {
 		args = append(args, "--output-dir", ".")
 	}
+	if len(ApiVersions) > 0 {
```
Comment (torstenwalter) [inline pkg/helmt/helmt.go:128]: not sure if this is needed

File: pkg/helmt/helmt.go [hunk 2]
```diff
@@ -124,6 +125,11 @@ func template(name string, chart string, values []string, namespace string, skip
...
+		for _, apiversion := range ApiVersions {
+			args = append(args, "--api-versions", apiversion)
```
Comment (torstenwalter) [inline pkg/helmt/helmt.go:130]: are you sure that you can specify api version flag multiple times?

`-a, --api-versions stringArray     Kubernetes api versions used for Capabilities.APIVersions`

Comment (vivian-src) [reply pkg/helmt/helmt.go:130]: specifying multiple time works but when I specify as key=value and string seperated by comma it doesn't work

Comment (vivian-src) [reply pkg/helmt/helmt.go:130]: these cases don't work
`--api-versions "monitoring.coreos.com/v1","monitoring.coreos.com/v1alpha1"`
`--api-versions 0="monitoring.coreos.com/v1",1="monitoring.coreos.com/v1alpha1"`

File: pkg/helmt/helmt.go (hunk 1 above)
Comment (vivian-src) [reply pkg/helmt/helmt.go:128]: In case nothing is specified for apiVersions in helm-chart.yaml, the block gets executed and leads to errors

**Review by torstenwalter — APPROVED** [review]

**Pull request merged**